Web/Assignment3/modalBootstrap.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="ISO-8859-1">
+<title>Question3</title>
+<link rel="stylesheet" href="css/bootstrap.css" type="text/css">
+<link rel="stylesheet" href="css/bootstrap-theme.css" type="text/css">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+<script src="js/bootstrap.min.js"></script>
+<script type="text/javascript">
+	$(document).ready(function() {
+		$('.ques').addClass('col-lg-1 col-md-1 col-sm-1 col-xs-1');
+		$('.quesText').addClass('col-lg-11 col-md-11 col-sm-11 col-xs-12');
```

Comment:
Don't add bootstrap Class in jquery if you want to use then use custom class and make it responsive using media query